### ISSUE TITLE
Css Module broken when ssr

### ISSUE THREAD

**Issue by JounQin** [vuejs/vue-loader#612]
deadly simple code:

``` vue
<template>
  <div :class="$style.home">
    {{ text }}
  </div>
</template>
<script>
  export default {
    data() {
      return {
        text: 'My Text'
      }
    }
  }
</script>
<style module>
  .home {
    color: red;
  }
</style>
```

When I'm trying to run it with ssr, there is an error:

``` bash
[Vue warn]: Error when rendering anonymous component at /local/GitHub/vue-ssr/src/views/Home/index.vue: 
events.js:165
      throw err;
      ^

Error: Uncaught, unspecified "error" event. (TypeError: Cannot read property 'home' of undefined)
```

I have to do this:
``` html
<div :class="$style && $style.home">
  {{ text }}
  <router-link to="/test">Go to test</router-link>
</div>
```

Then there will be no error, but after server rendered at first it will be an empty className and then when all *.js* have been resolved, the page will refresh automatically with correct className.

**Comment by yyx990803** [comment]
Can't reproduce... copied your component into a SSR project and it works as intended. Please provide a reproduction.

**Comment by JounQin** [comment]
https://github.com/JounQin/vue-ssr

I'm using SSR with koa2, CSS Modules component: *src/views/Home*, clone project and run `yarn && yarn dev`, then open `localhost:3000`, it will show the error message.

**Comment by yyx990803** [comment]
Ah, this happens to have been fixed in 10.2.0.

**Comment by JounQin** [comment]
😂 Yes, it works in 10.2.0.
